src/evaluate.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import numpy as np
+import re
+import time
+import json
+from src.model import YoModel
+from tqdm import tqdm
+from typing import List, Tuple, Union
+from src.utils import WORDS_REGEX
+from dataclasses import dataclass, asdict
+from sklearn.metrics import accuracy_score, precision_score, recall_score, \
+    f1_score, roc_auc_score
+
+
+@dataclass
+class EvaluateResult:
+    """Model evaluation result."""
+    accuracy: float
+    precision: float
+    recall: float
+    f1_score: float
+    auroc: float
+    wall_time: float
+    cpu_time: float
+    token_to_sec: float
```

Comment:
Кажется токены в секунду корректнее вот так записать:
```suggestion
    tokens_per_sec: float
```